_posts/en/newsletters/2020-07-01-newsletter.md

```diff
@@ -0,0 +1,361 @@
+    encounters segwit transactions that are invalid because of
+    accidental mistakes.  It may also complicate the development of new
+    relay protocols such as [package relay][topic package relay].
+
+    The long-term solution to the problem is that transactions should be
+    announced using a hash that commits to the entire transaction---both
+    the legacy fields and the new segwit fields.  That's exactly what
+    wtxids do, and they're already needed in the protocol to construct
+    the witness merkle root in each block's coinbase transaction.  This
+    new BIP proposes updating the P2P protocol's `inv` message that
+    announces new transactions, and the `getdata` message that requests
+    a transaction, to allow them both to use wtxids.  That will allow
+    nodes to skip re-downloading a transaction if it has the same wtxid
+    as a transaction that was previously found to be invalid or
+    unacceptable.
+
+    The BIP339 proposal also adds the additional feature negotiation
+    between newly started nodes described in [Newsletter #87][news87
+    negotiation].  See also the [proposed implementation][Bitcoin Core
+    #18044] for Bitcoin Core.
+
+- [BIPs #923][] adds the [BIP78][] specification for the version of the
+  [payjoin][topic payjoin] protocol originally implemented in BTCPay
+  (see [Newsletter #94][news94 btcpayjoin]).  Payjoin allows a spender
+  and a receiver to both add UTXOs to a transaction.  This decreases the
+  reliability of the assumption made by third-party block chain
+  surveillance companies that any set of UTXOs spent in the same
+  transaction were all received by the same person.  BIP78 is based on
+  the [BIP79][] specification of the Bustapay payjoin variant (see
+  [Newsletter #13][news13 bustapay]) but contains several notable
+  differences, including different extensions to [BIP21][] `bitcoin:`
+  URIs, usage of PSBTs, and some additional requirements designed to
+  enhance privacy.  Several programs already support this protocol and
+  several more are currently working on adding support.  The PR for this
+  BIP received significant discussion and may provide useful reference
+  material for anyone interested in the subject.
+
+- [BIPs #550][] revises the [BIP8][] alternative to [BIP9][]
+  versionbits-based soft fork deployment.  BIP8 previously allowed a
+  soft fork to be activated by miner signaling using the same parameters
+  as would be used for BIP9, but it required that the soft fork activate
+  at the end of the signaling period even if miners were still not
+  signaling readiness to follow the new consensus rules.  This could be
+  used to override miners who were obstructing activation of a fork, but
+  it could also lead to diverging block chains between nodes that
+  enforced the fork's new consensus rules and those that didn't.
+
+    The main change to BIP8 from its previous version is that it
+    may now be used initially without requiring mandatory
```

Comment:
initially used?

Reply:
I think this is the best way to say it, but that's just my instinct. :man_shrugging: